src/view/details.js

```diff
@@ -0,0 +1,131 @@
+define(["text!../template/details.html"], function (html) {
+
+    var body = document.body;
+
+    // Load the HTML
+    var template = document.createElement("body");
+    template.innerHTML = html;
+
+    // Itinerary
+    var title = template.querySelector("h2");
```

Comment:
No worries.

--

Ismael Stahelin, Software Engineer
ArcTouch - App Development Studio

ismael.stahelin@arctouch.com
+1-415-944-2000 main

twitter.com/arctouch

Custom apps for world-class brands and the Fortune 500
arctouch.com/work | arctouch.com/blog

On Mon, Oct 5, 2015 at 3:57 PM, Diego Weiss <notifications@github.com>
wrote:

> In src/view/details.js
> <https://github.com/diegolw/floripaByBus/pull/1#discussion_r41208494>:
>
> > @@ -0,0 +1,131 @@
> > +define(["text!../template/details.html"], function (html) {
> > +
> > +    var body = document.body;
> > +
> > +    // Load the HTML
> > +    var template = document.createElement("body");
> > +    template.innerHTML = html;
> > +
> > +    // Itinerary
> > +    var title = template.querySelector("h2");
>
> Good! When I started to do it I thought in avoid to use jQuery, but I
> tested the request with jQuery and I forget to remove it after, converting
> to XMLHttpRequest. Was lack of attention, my fault again!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diegolw/floripaByBus/pull/1/files#r41208494>.
>